src/main/java/com/umc/networkingService/domain/todayILearned/mapper/TodayILearnedMapper.java

```diff
@@ -0,0 +1,58 @@
+package com.umc.networkingService.domain.todayILearned.mapper;
+
+import com.umc.networkingService.domain.member.entity.Member;
+import com.umc.networkingService.domain.todayILearned.dto.requeest.TodayILearnedRequest.TodayILearnedCreate;
+import com.umc.networkingService.domain.todayILearned.dto.requeest.TodayILearnedRequest.TodayILearnedUpdate;
+import com.umc.networkingService.domain.todayILearned.dto.response.TodayILearnedResponse.TodayILearnedId;
+import com.umc.networkingService.domain.todayILearned.dto.response.TodayILearnedResponse.TodayILearnedInfo;
+import com.umc.networkingService.domain.todayILearned.dto.response.TodayILearnedResponse.TodayILearnedInfos;
+import com.umc.networkingService.domain.todayILearned.entity.TodayILearned;
+import com.umc.networkingService.domain.todayILearned.entity.TodayILearnedFile;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TodayILearnedMapper {
+
+    public TodayILearned toTodayILearned(Member member, TodayILearnedCreate request) {
+        return TodayILearned.builder()
+                .writer(member)
+                .title(request.getTitle())
+                .subtitle(request.getSubTitle())
+                .content(request.getContent())
+                .part(request.getPart())
+                .build();
+    }
+
+    public TodayILearnedId UUIDtoTodayILearnedId(UUID todayILearnedId) {
```

Comment:
다른 함수들과 통일성을 위해서 함수명을 다음과 같이 변경하면 좋을 거 같아요
```suggestion
    public TodayILearnedId toTodayILearnedId(UUID todayILearnedId) {
```

Reply:
수정 완료